### ISSUE TITLE
Refactor gen_phrase() to allow for non-SVO languages

### ISSUE THREAD

**Issue by mkzero** [Bytespeicher/twitterstatus#8]
Some languages don't work with the current SVO word ordering used by the gen_phrase function. The function should take another optional argument from `wordlist` that makes it use a different word ordering. https://github.com/Bytespeicher/twitterstatus/blob/b423638e8b533b5d81fd3408300b153f8080a249/twitterstatus.py#L40

**Comment by randName** [comment]
The way I can see this working is to add a format string to the `translations` dictionary that determines the order in which the words will be joined.

However, I don't see the pressing need for this change as your sentence at this point is rather simple. In Japanese (the only non-SVO language I personally know), a possible sentence will be "[ Space ] [ は ] [ （営業中｜休み中）。 ] [ Extra stuff ]". While there are nuances in exactly which part of the sentence does what, the structure is still pretty similar.

**Comment by mkzero** [comment]
@randName true, most languages actually use the same ordering but there might be other problems like RTL instead of LTR composition, which would make the format string a valuable addition.
